navigators/RootStackNavigator.tsx

```diff
@@ -1,6 +1,6 @@
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
 import LoadingScreen from '../screens/LoadingScreen';
-import LoginScreen from '../screens/LoginScreen';
+import { LoginScreen } from '../screens/LoginScreen';
```

Comment:
```suggestion
import LoginScreen from '../screens/LoginScreen';
```
_(Check suggestion in LoginScreen.tsx first)_

Reply:
didnt work that way

Reply:
It should work if the LoginScreen function is the default export, otherwise it won't ^^